src/core/lib/iomgr/fork_posix.cc

```diff
@@ -41,47 +41,57 @@
  *       AROUND VERY SPECIFIC USE CASES.
  */
 
+static bool skipped_handler = true;
```

Comment:
Can you use anonymous namespace instead since this is new code?

Reply:
Can I put these in an anonymous namespace and then use them from non-anonymous functions below?

Reply:
Turns out I can, even without the wrapping ```namespace grpc_core``` (which I want to avoid because these functions are on the surface)